common/app/common/utils/Translator.java

```diff
@@ -0,0 +1,29 @@
+package common.utils;
+
+import io.sphere.sdk.models.LocalizedStrings;
+
+import java.util.List;
+import java.util.Locale;
+
+public class Translator {
+
+    final Locale userLanguage;
+    final List<Locale> userFallbackLanguages;
+    final List<Locale> projectLanguages;
+
+    private Translator(final Locale userLanguage, final List<Locale> userFallbackLanguages, final List<Locale> projectLanguages) {
+        this.userLanguage = userLanguage;
+        this.userFallbackLanguages = userFallbackLanguages;
+        this.projectLanguages = projectLanguages;
+    }
+
+    public static Translator of(final Locale userLanguage, final List<Locale> userFallbackLanguages, final List<Locale> projectLanguages) {
+        return new Translator(userLanguage, userFallbackLanguages, projectLanguages);
+    }
+
+    public String translate(final LocalizedStrings localizedStrings) {
```

Comment:
It would be nice to have Javadoc here. It looks like you return an empty String on not found, right?